app/javascript/components/ansible-repository-form/ansible-repository-form.schema.js

```diff
@@ -38,15 +39,10 @@ function createSchema(repositoryId) {
       id: 'scm_url',
       name: 'scm_url',
       isRequired: true,
-      validate: [{
-        type: validatorTypes.REQUIRED,
-        message: __('Required'),
-      },
-      {
-        type: validatorTypes.URL,
-        message:
-        __('URL must include a protocol (http://, https:// or file://) or be a valid SSH path (user@server:path or ssh://user@address:port/path)'),
```

Comment:
Why are we removing helper text?

Reply:
We're not removing it it's still here https://github.com/ManageIQ/manageiq-ui-classic/pull/8164/files#diff-c85ae366cfc655ca386d42b3e8fb06a40f46df04a86fff605a984308fad63cdbR45